sdk/core/scheduler/main.cc

```diff
@@ -258,6 +258,15 @@ namespace sched
 	                                               size_t        mepc,
 	                                               size_t        mtval)
 	{
+		if constexpr (DebugScheduler)
+		{
+			/* Ensure that we got here from an IRQ-s deferred context */
+			auto returnAddress = Capability{__builtin_return_address(0)};
```

Comment:
I would write this as:

```suggestion
			Capability returnAddress{__builtin_return_address(0)};
```

But I don't think it matters too much.